src/main/java/core/basesyntax/ShopServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import core.basesyntax.operations.OperationHandler;
+import core.basesyntax.operations.OperationStrategy;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ShopServiceImpl implements ShopService {
+    private final Map<String, Integer> storage = new HashMap<>();
```

Comment:
The `storage` Map should be in its own `db` package as per the checklist requirement to keep logic separated in different packages. This will help in maintaining a clean structure of the code.

Reply:
The storage Map should be in its own db package as per the checklist requirement to keep logic separated in different packages. This will help in maintaining a clean structure of the code.